src/averageAges.js

```diff
@@ -14,12 +14,16 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mans = people.filter((person) => person.sex === 'm');
```

Comment:
Use correct English.
```suggestion
  const men = people.filter((person) => person.sex === 'm');
```